tenprintcover.py

```diff
@@ -32,6 +32,9 @@
 
 import cairocffi as cairo
 
+PY2 = sys.version_info[0] == 2
+if PY2:
+    FileNotFoundError = IOError
```

Comment:
Perhaps we should remove these three lines from this PR, as they are discussed in issue https://github.com/mgiraldo/tenprintcover-py/issues/5 separately.